tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -83,7 +83,7 @@ def test_multiple_jobs_submission(scheduler, region, pcluster_config_reader, clu
 @pytest.mark.instances(["c5.xlarge"])
 @pytest.mark.schedulers(["sge", "torque"])
 @pytest.mark.oss(["alinux2", "centos7", "ubuntu1804"])
-@pytest.mark.usefixtures("region", "instance")
+@pytest.mark.usefixtures("region", "instance", "os")
```

Comment:
no @pytest.mark.oss only applies filtering but the actual fixture is not automatically injected. This test was not being executed because it was expecting os to be one of ["alinux2", "centos7", "ubuntu1804"] while it was unset.
The alternative is to automatically inject the dimension fixtures (region, os, instance, scheduler) in the test cases but this has the downside that you can't really have a test that does not depend on one of the dimensions.